build_files/cache_kernel.sh

```diff
@@ -3,8 +3,10 @@
 set -eoux pipefail
 
 if [[ -n "${NVIDIA_TYPE:-}" ]]; then
```

Comment:
This variable is only set for gts/stable.

Reply:
well, that's part of the problem then... it's getting set for latest with the last PR which got merged...

Reply:
I think this is good. If not, well, lets do another PR. :-)